SPECS/intel-igc.spec

```diff
@@ -37,6 +37,9 @@ Requires: kernel-uname-r = %{kernel_version}
 Requires(post): /usr/sbin/depmod
 Requires(postun): /usr/sbin/depmod
 
+# This RPM obsoletes XCP-ng specific RPM igc-module
+Obsoletes: igc-module
```

Comment:
AFAIK it is better to version the Obsoletes in order to keep the possibility to bring the old RPM back with a higher release.

Reply:
Added